src/components/copy-text-button.tsx

```diff
@@ -40,7 +40,6 @@ export function CopyTextButton( {
 		}
 		setTimeoutId( setTimeout( () => setShowCopied( false ), timeoutConfirmation ) );
 	}, [ text, timeoutConfirmation, timeoutId ] );
-	console.log( 'Icon Size:', iconSize );
```

Comment:
Thanks for removing this!

Reply:
@katinthehatsite sure thing! Are you comfortable approving the entirety of this pull request? Or would you prefer I seek input from others as well?

Reply:
Approved! I got sidetracked for a moment but all good